server/settings/publish.py

```diff
@@ -159,6 +175,15 @@ class PublishPluginsModel(BaseSettingsModel):
         "active": True,
         "families" : []
     },
+    "CollectFramesFixDefHou": {
+        "enabled": True,
+        "optional": True,
```

Comment:
@MustafaJafar does it makes sense to have this optional? :) The plug-in itself doesn't support optional state yet but that'd be easy to implement.